tests/h/services/group_members_test.py

```diff
@@ -196,7 +196,7 @@ def test_provides_realtime_publisher_as_publish(self, patch, pyramid_request):
 
 
 @pytest.fixture
-def usr_svc(pyramid_request, db_session):
+def usr_group_members_service(pyramid_request, db_session):
```

Comment:
I still think the name is surprising because this fixture isn't standing in for a service but rather a function which fetches users given their ID, which is called `user_fetcher` in the `GroupMembersService.__init__` definition. This function just happens to be a bound method on a service in the actual app. Calling this just `user_fetcher` would be fine.

Reply:
This seems to me like it belongs inside the `group_members_service` fixture below, which looks like the only place it's used, unless it needs to be its own separate fixture for some reason? If you also make it a mock then you'll be able to assert things about how it has been called, easily change its return value, or make it raise exceptions:

```python
@pytest.fixture
def group_members_service(db_session, publish):
    def fetch(userid):
        return db_session.query(User).filter_by(userid=userid).one_or_none()
    user_fetcher = mock.create_autospec(fetch, side_effect=fetch)
    return GroupMembersService(db_session, user_fetcher, publish=publish)
```